docs/set-of-experiments.rst

```diff
@@ -0,0 +1,36 @@
+==============
+Working with a set of experiments
+==============
+
+To keep the source code consistent while working with 
+a set of ``experiments`` (several ``benchmark/ProgrammingModels`` x ``systems``),
+one can consider working with a single ``Benchpark workspace``, which will ensure the same
+clone of Spack, and the same clone of Ramble, will be used for all of the ``experiments`` in 
+this set.  To work with a set of ``experiments`` based on a single clone of Spack and 
+a single clone of Ramble, set up your experiments in the same /output/path/to/workspace_root::
```

Comment:
We might want to open with why a user would want to keep the same spack/ramble for multiple experiments

```
You may want to use the same workspace directory when setting up multiple experiments:

* Benchpark only stores one copy of Spack/Ramble per workspace, and does not repeatedly
  download them when reusing a workspace
* Since each experiment in the workspace shares the Spack/Ramble instances, then edits you
  you make to Spack packages will apply to all experiments in the workspace
```